lib/qmp.ml

```diff
@@ -180,6 +194,8 @@ let message_of_string str =
     | "xen-load-devices-state"   -> json |> xen_load_devices_state   |> fun x -> Xen_load_devices_state x
     | "xen-set-global-dirty-log" -> json |> xen_set_global_dirty_log |> fun x -> Xen_set_global_dirty_log x
     | "blockdev-change-medium"   -> json |> blockdev_change_medium   |> fun (x, y) -> Blockdev_change_medium (x, y)
+    | "device_del"               -> json |> device_del               |> fun x -> Device_del x
+    | "qom_list"                 -> json |> qom_list                 |> fun x -> Qom_list x
```

Comment:
Here do you mean `qom-list` or `qom_list`

Reply:
oh, sorry for the error command , it is qom-list.

Reply:
updated